### PR TITLE
feat: add typos-lsp

### PR DESCRIPTION
**Pull request by mehalter**

### CODE REVIEW

**Comment by mehalter** [comment]
I believe I got the architectures specified correctly. Let me know if things like `linux_x64_gnu` should simply be `linux_x64`.

File: packages/typos-lsp/package.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+name: typos-lsp
+description: Source code spell checker
+homepage: https://github.com/tekumara/typos-vscode
+licenses:
+  - MIT
+  - Apache-2.0
```
Comment (williamboman) [inline packages/typos-lsp/package.yaml:7]: I can't see Apache-2.0 being referred to as their license anywhere

**Review by williamboman — APPROVED** [review]
LGTM! Just one question re: licenses

File: packages/typos-lsp/package.yaml (hunk 1 above)
Comment (mehalter) [reply packages/typos-lsp/package.yaml]: Oh yeah that's totally a mistake, copy/paste error. My bad! Resolved in the latest commit

**Pull request merged**